From a3b3b71f2d2fe7879fb9e444e70b2d74f0269fe1 Mon Sep 17 00:00:00 2001
From: Stefan Eilemann <Stefan.Eilemann@epfl.ch>
Date: Sat, 27 Dec 2014 14:44:16 +0100
Subject: [PATCH] Compile fix for Boost 1.57

---
 lunchbox/any.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lunchbox/any.h b/lunchbox/any.h
index 9f91e40..3618f3a 100644
--- a/lunchbox/any.h
+++ b/lunchbox/any.h
@@ -38,6 +38,7 @@
 #include <boost/serialization/access.hpp>
 #include <boost/serialization/assume_abstract.hpp>
 #include <boost/serialization/base_object.hpp>
+#include <boost/serialization/type_info_implementation.hpp>
 #include <boost/serialization/shared_ptr.hpp>
 
 // See boost/python/type_id.hpp
